pegasus/forms/student_nomination.rb

```diff
@@ -1,20 +0,0 @@
-class StudentNomination
```

Comment:
Note that, for both `StudentNomination` and `TeacherNomination`, it would be worth keeping these files if they contained a `process` method in the offset chance we needed to reprocess any of the existing data.

As `normalize` and `receipt` are used at creation time only, it seems safe to remove them.